src/Components/Register/Register.js

```diff
@@ -51,29 +77,93 @@ const Register = ({ onSubmit, errorLabel }) => {
   repeatPasswordLabel   = registerTranslations.repeatPassword,
   alterEgoLabel         = registerTranslations.alterEgo,
   iconLabel             = registerTranslations.icon,
-  submitLabel           = registerTranslations.submit;
+  submitLabel           = registerTranslations.submit,
+  pwdTooltipLabel       = registerTranslations.passwordTooltip,
+  repeatPwdTooltipLabel = registerTranslations.repeatPasswordTooltip,
+  emailTooltipLabel     = registerTranslations.emailTooltip,
+  userTooltipLabel      = registerTranslations.userTooltip,
+  alterEgoTooltipLabel  = registerTranslations.alterEgoTooltip
+
+  let pwdError = null;
+  if(isPwdMatched === true){
+    pwdError = styles.registerinput
+  }else{
+    pwdError = styles.registerinputError
+  }
 
   return (
     <div className={styles.registerContainer} onClick={e => e.stopPropagation()}>
       <h2 className={styles.registerTitle}>{titleLabel}</h2>
       <form onSubmit={handleSubmit} className={styles.registerForm}>
         { errorLabel && <h3 className={styles.registerError}>{errorLabel}</h3>}
         <h3 className={styles.registerHeadline}>{emailLabel}</h3>
-        <input name="email" className={styles.registerinput} value={email} onChange={handleChange} />
+        <input
+          type="email"
+          name="email"
+          required
+          value={email}
+          minLength="7"
```

Comment:
Please move this magic numbers to constants file.

Reply:
Done.